subsys/usb/device/class/cdc_acm.c

```diff
@@ -230,7 +230,7 @@ static void cdc_acm_write_cb(uint8_t ep, int size, void *priv)
 	 * ensure that actual payload will not be sent before initialization
 	 * timeout passes.
 	 */
-	if (ring_buf_is_empty(dev_data->tx_ringbuf) && size) {
+	if (ring_buffer_empty(dev_data->tx_ringbuf) && size) {
```

Comment:
These are pointless renamings that break the API, change the code, break all the patches and OOT code for nothing. They are not acceptable. And your changes do not follow https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#pr-requirements, please get more familiar with development process first.